static/js/services/live-list.js

```diff
@@ -48,27 +51,37 @@ angular.module('inboxServices').factory('LiveListConfig',
           }
           return (c1.name || '').toLowerCase() < (c2.name || '').toLowerCase() ? -1 : 1;
         },
-        listItemForTemplate: function() {
-          return function(contact) {
-            var template = $templateCache.get('templates/partials/contacts_list_item.html');
-            var scope = $scope.$new();
-            scope.contact = contact;
-            scope.primaryContactName = { name: contact.contact };
-            return evaluateExpressions(template, scope);
-          };
+        icons: {
+          person: 'fa-user',
+          clinic: 'fa-group',
+          health_center: 'fa-hospital-o',
+          district_hospital: 'fa-building'
+        },
```

Comment:
Should this not be pulled from the contact schema? Feels like a duplication

(feel free to nuke / rework the contact schema, it's a pretty weird but self-contained piece of code)

Reply:
Oh man, you know what happened? I forgot ContactSchema existed. Yes, that's much better. I'm not going to spend time refactoring it now although I agree it needs it, but it might be eliminated completely with #3779 #3570 and/or #3639